site/src/atomics/border.md

```diff
@@ -118,3 +118,20 @@ The color of a border can be modified using the following classes.
 	<p class="margin-inline-md">Custom Color</p>
 </div>
 ```
+
+### Gradient colors
+
+The gradient colors of a border can be modified using the following classes.
+
+| modifiers | color name         | direction |
```

Comment:
Super nit, please feel free to resolve as a wontfix:

```suggestion
| modifiers | gradient name         | direction |
```

Reply:
Updated